charts/keycloakx/values.yaml

```diff
@@ -275,15 +275,15 @@ ingress:
       host: '{{ .Release.Name }}.keycloak.example.com'
       # Paths for the host
       paths:
-        - path: /
+        - path: '{{ trimSuffix "/" .Values.http.relativePath}}/'
           pathType: Prefix
   # TLS configuration
   tls:
     - hosts:
         - keycloak.example.com
       secretName: ""
 
-  # ingress for console only (/auth/admin)
+  # ingress for console only ({{ trimSuffix "/" .http.relativePath}}/admin)
```

Comment:
I think I would not include this in the comment.

Reply:
Agree, in comment it looks strange.